<!-- 
This launch file starts soundplay_node.py, which is the node that plays
sounds based on messages on the robotsound topic.  
-->

<launch>
  <node name="soundplay_node" pkg="sound_play" type="soundplay_node.py" required="true"/>
  <node name="KomodoSpeech" pkg="komodo_speech" type="komodospeech.py" output="screen" respawn="true" cwd="node" />

</launch>

